pages/linux/ldd.md

```diff
@@ -1,12 +1,25 @@
 # ldd
 
-> Display shared library dependencies.
+> Display shared library dependencies of a binary.
+> Do not use on an untrusted binary, use objdump for that instead.
 > More information: <https://manned.org/ldd>.
 
 - Display shared library dependencies of a binary:
 
 `ldd {{path/to/binary}}`
 
+- Display _all_ information about dependencies:
```

Comment:
I'd be surprised if that worked in even half of the tldr clients

Reply:
My mistake, I can remove it if you would like?